results.go

```diff
@@ -129,6 +129,16 @@ type Link struct {
 	Rel  string `mapstructure:"rel"`
 }
 
+// PrettyPrintJSON creates a string containing the links as
+// pretty-printed JSON. It is useful for displaying the results.
+func (link Link) PrettyPrintJSON() string {
```

Comment:
@jrperritt that will format the entire result as JSON; but there are cases where we don't want all the result as JSON, only the links. e.g. https://github.com/rackspace/rack/pull/346/files#diff-89ed2b55dee1ba0cb9742aba85a8f6deR58